examples/insurance/index.rsh

```diff
@@ -0,0 +1,270 @@
+        ])], Null),
+        notifyMembersAboutNewClaim: Fun([Struct([
+            ["ownerAddr", Address],
+            ["amountRequested", UInt],
+            ["description", Bytes(600)],
+            ["supportDocuments", Bytes(100)]
+        ])], Null),
+        seeFeedback: Fun([], Null),
+        signout: Fun([], Null),
+        notifyFundedMember: Fun([Address], Null),
+        stopContract: Fun([], Null),
+        log: Fun(true, Null) //REF: https://docs.reach.sh/guide/logging/
+    });
+
+    const CommunityMember = API('CommunityMember', {
+        registerMembership: Fun([Struct([
+            ["fullName", Bytes(60)], ["phone", Bytes(20)],
+            ["email", Bytes(60)],
+            ["chosenInsurancePackage", UInt]
+        ])], Bool),
+        payMonthlyFee: Fun([Struct([["who", Address], ["mfee", UInt]])], Bool),
+        createClaim: Fun([Struct([
+            ["amountRequested", UInt], ["amountSet", UInt], ["accepted", Bool],
+            ["approvalsCount", UInt], ["sumOfSetAmounts", UInt],
+            ["insrPackageId", UInt], ["amountDue", UInt], ["matureBalance", UInt],
+            ["fundLimit", UInt], ["description", Bytes(200)]
+        ])], Bool),
+        respondToClaim: Fun([Struct([
+           ["claimant", Address], ["accepted", Bool], ["setAmount", UInt]
+        ])], Bool),
+        withDrawClaim: Fun([], Bool),
+        //changePackage: Fun([Bytes(60)], Bool),
+        stopContract: Fun([], Bool)
+    });
+    setOptions({ untrustworthyMaps: true });
+    init();
+
+
+    //REF: REACH ARCHITECTURE: https://docs.reach.sh/rsh/#ref-programs
+
+    Insurer.only(() => {
+        const mandatoryEntryFee = declassify(interact.mandatoryEntryFee);
+        const contractIsRunning = declassify(interact.contractIsRunning);
+        interact.seeFeedback();
+    });
+    Insurer.publish(mandatoryEntryFee, contractIsRunning);
+    const invariantCondition = true;
+    Insurer.interact.log("backend: starting...");
+    commit();
+    Insurer.publish();
+
+    //keep a list of all members' Addresses,
+    //more info about the members is kept away (in the db).
+    const registeredMembers = new Set();
+
+    //claim shape
+    const insuranceClaims = new Map(Struct([
+      ["amountRequested", UInt], ["amountSet", UInt], ["accepted", Bool],
+      ["approvalsCount", UInt], ["sumOfSetAmounts", UInt]
+    ]));
+
+    //details of members with open claims are kept close, 
+    //other members are kept away from here (in the db)
+    const claimOwners = new Map(Struct([
+        //["fullName", Bytes(60)],
+        //["physicalAddress", Bytes(100)],
+        //dateJoined, Bytes(30),
+        ["insrPackageId", UInt],
+        ["amountDue", UInt],
+      ["matureBalance", UInt]
+    ]));
+
+
+    const [
+        membersCount,
+        claimsCount
+    ] = parallelReduce([1, 1])
+        .define(() => {
+            //the "readFromMap" function below can be passed to maybe(...) function to 
```

Comment:
Clean up codebase for publication. Remove lengthy comments. Integrate them into the guide if they're relevant.